File: pset6/mario/mario.py
from cs50 import get_int

h = get_int("Height: ")
while h < 1 or h > 8:
    h = get_int("Height: ")
    
for i in range(h):
    print(" " * (h - i - 1), end="")
    print("#" * (i + 1), end="  ")
    print("#" * (i + 1))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h = get_int("Height:");
    while (h <= 0 || h > 8)
    {
        h = get_int("Height:");
    }

    // print blocks of mario
    for (int i = h - 1; i >= 0; i--)
    {
        for (int k = 0; k < i; k++)
        {
            printf(" ");
        }
        for (int j = 0; j < h - i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < h - i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}